charts/lws/README.md

```diff
@@ -0,0 +1,67 @@
+# lws's helm chart
+
+## Table of Contents
+
+<!-- toc -->
+- [Installation](#installation)
+    - [Prerequisites](#prerequisites)
+    - [Installing the chart](#installing-the-chart)
+        - [Install chart using Helm v3.0+](#install-chart-using-helm-v30)
+        - [Verify that controller pods are running properly.](#verify-that-controller-pods-are-running-properly)
+    - [Configuration](#configuration)
+<!-- /toc -->
+
+### Installation
+
+Quick start instructions for the setup and configuration of lws using Helm.
+
+#### Prerequisites
+
+- [Helm](https://helm.sh/docs/intro/quickstart/#install-helm)
```

Comment:
Thank you for your comment! I believe that Cert-manager should not be listed as a prerequisite for the Helm chart since it can be installed optionally if the user sets `enableCertManager: true` in `values.yaml`. In this context, prerequisites should only include Helm itself, while Cert-manager is treated as an application dependency that users can choose to enable.

Reply:
That's why it's optional